src/modules/UI/scenes/Request/styles.js

```diff
@@ -11,7 +11,7 @@ const styles = StyleSheet.create({
   view: {
```

Comment:
This PR is for the FlipInput. What changes were needed in this file for the FlipInput animation?